access-functions.php

```diff
@@ -25,16 +25,35 @@ function graphql_format_field_name( $field_name ) {
 /**
  * Provides a simple way to run a GraphQL query with out posting a request to the endpoint.
  *
- * @param string $request        The GraphQL query to run
+ * @param array $request_data The GraphQL request data (query, variables, operation_name).
+ *
+ * @access public
+ * @return array
+ * @since  0.1.4
+ */
+function graphql( $request_data = [] ) {
```

Comment:
Does this support batch requests? Like:

```
$data = graphql([
  [
    'query' => '{posts{edges{node{id,title}}}}'
  ],
  [
    'query' => '{morePosts:posts{edges{node{id,title}}}}'
  ]
]);
```

Reply:
Should have just checked. It looks like it does not. 🤔 

Not sure it _needs_ to though 🤔

Reply:
So I just checked and this doesn't support batch requests. Though I'm not fully sure it really _needs_ to, because the server can execute many `graphql()` calls in a single page-load and it would be the same as batch requesting. . .so I think it's a non-issue.